driver-core/build.gradle

```diff
@@ -89,7 +89,7 @@ afterEvaluate {
             'com.github.luben.zstd.*;resolution:=optional',
             'org.slf4j.*;resolution:=optional',
             'jnr.unixsocket.*;resolution:=optional',
-            'com.mongodb.crypt.capi.*;resolution:=optional',
```

Comment:
driver-core doesn't reference `MongoCryptException` (the only class left in this package), so this needed to be removed or else we get a warning.